tests/acceptance/test_project_settings_sampling.py

```diff
@@ -103,27 +103,30 @@ def test_add_uniform_rule_with_recommended_sampling_values(self):
                 == serializer.validated_data["rules"][0]
             )
 
-    @pytest.mark.skip(reason="Flaking pretty consistently")
     def test_add_uniform_rule_with_custom_sampling_values(self):
         with self.feature(FEATURE_NAME):
             self.wait_until_page_loaded()
 
             # Open uniform rate modal
-            self.browser.element('[aria-label="Start Setup"]').click()
+            self.browser.click_when_visible('[aria-label="Start Setup"]')
+
+            self.browser.wait_until('[id="recommended-client-sampling"]')
 
             # Enter a custom value for client side sampling
-            self.browser.element('[id="recommended-client-sampling"]').clear()
-            self.browser.element('[id="recommended-client-sampling"]').send_keys(80, Keys.ENTER)
+            self.browser.element('[id="recommended-client-sampling"]').send_keys(
+                Keys.BACK_SPACE, Keys.BACK_SPACE, Keys.BACK_SPACE, 80, Keys.ENTER
```

Comment:
This does not seem like good practice to me, why are you calling back space three times here?

Reply:
yeah, i can remove it back to original version.

Reply:
please do then, and this one too 7ff3361ad86b28f71d6915a47fe070a633feec806854b92a90d0fd0d3a8e2403R122 ?